Assets/HoloToolkit/Utilities/Scripts/DisplayDependentObjectActivator.cs

```diff
@@ -7,22 +7,25 @@
 using UnityEngine.XR.WSA;
 #endif
 
-public class DisplayDependentObjectActivator : MonoBehaviour
+namespace HoloToolkit.Unity
```

Comment:
Are any of the others under .Utilities?

Reply:
No, they're all under `HoloToolkit.Unity`